### PR TITLE
Add panel button, performance metrics, new logo, misc fixes

### PR DESCRIPTION
**Pull request by tjcouch-sil**

### CODE REVIEW

File: react-electron-poc/src/renderer/components/layout/Layout.tsx [hunk 1]
```diff
@@ -100,134 +242,212 @@ const Layout = () => {
...
-                    placeholderText: 'Loading zzz6 Psalm 119 USX',
-                    textPromise: getScripture('zzz6', 19, 119),
-                } as TextPanelProps,
-                {
-                    title: 'zzz6: Psalm 119 USX',
-                },
-            ); */
+
+            /* // Add the default tabs
+            if (panelManager.current.panelsInfo.size >= startingTabs) return;
```
Comment (katherinejensen00) [inline react-electron-poc/src/renderer/components/layout/Layout.tsx:249]: Is this comment still needed? Where does it end?

File: react-electron-poc/src/renderer/components/layout/Layout.tsx [hunk 2]
```diff
@@ -100,134 +242,212 @@ const Layout = () => {
...
+                } as ScriptureTextPanelStringProps,
                 {
                     position: {
                         direction: 'below',
                         referencePanel: ohebPanel.id,
                     },
                 },
             );
+
+            /* if (startingTabs < 6) return;
```
Comment (katherinejensen00) [inline react-electron-poc/src/renderer/components/layout/Layout.tsx:343]: Is this comment still needed?

File: react-electron-poc/src/renderer/components/layout/Layout.tsx [hunk 3]
```diff
@@ -100,134 +242,212 @@ const Layout = () => {
...
+                            checked={browseBook}
+                            onChange={(event) =>
+                                updateBrowseBook(event.target.checked)
+                            }
+                        />
+                    </label>
                 </span>
             </div>
             <div className="layout-dock">
+                {/* <ScriptureTextPanelSlate
```
Comment (katherinejensen00) [inline react-electron-poc/src/renderer/components/layout/Layout.tsx:437]: Are you leaving this comment there as an option that you can come back to as needed?

File: react-electron-poc/src/renderer/components/panels/TextPanels/ScriptureTextPanelHtml.tsx [hunk 4]
```diff
@@ -20,240 +24,266 @@ import './TextPanel.css';
...
-                            node = node.lastChild as ChildNode;
-                        }
-                    } else {
-                        // This is the first node of its siblings, so get the parent and try again
-                        node = node.parentNode as ParentNode;
+                if (selectedChapter >= 0) {
+                    // We got our results! Done
+                    break;
+                } else if (node.previousSibling) {
+                    // This node has a previous sibling. Get the lowest node of the previous sibling and try again
```
Comment (katherinejensen00) [inline react-electron-poc/src/renderer/components/panels/TextPanels/ScriptureTextPanelHtml.tsx:129]: These comments are helpful. Thanks for inserting them!

File: react-electron-poc/src/renderer/components/panels/Erb/Erb.css [hunk 5]
```diff
@@ -2,7 +2,7 @@
  * @NOTE: Prepend a `~` to css file paths that are in your node_modules
  *        See https://github.com/webpack-contrib/sass-loader#imports
  */
-.wrapper {
+.erb-wrapper {
```
Comment (katherinejensen00) [inline react-electron-poc/src/renderer/components/panels/Erb/Erb.css:5]: I hadn't seen erb before, but when I googled erb it looked really cool and helpful. Nice work!

**Review by katherinejensen00 — APPROVED** [review]
Great work, TJ! The different types of panels are really nice and there is a good setup for write apis. The icons are also pretty cool. Way to go!

File: react-electron-poc/src/renderer/components/layout/Layout.tsx (hunk 3 above)
Comment (tjcouch-sil) [reply react-electron-poc/src/renderer/components/layout/Layout.tsx:437]: Yep! I think this one may be worth leaving in if that's ok - doing this eliminates some variables when trying to optimize slate :)

File: react-electron-poc/src/renderer/components/layout/Layout.tsx (hunk 1 above)
Comment (tjcouch-sil) [reply react-electron-poc/src/renderer/components/layout/Layout.tsx:249]: Good call. I will take it out :) thanks!

File: react-electron-poc/src/renderer/components/layout/Layout.tsx (hunk 2 above)
Comment (tjcouch-sil) [reply react-electron-poc/src/renderer/components/layout/Layout.tsx:343]: Good call. I will take it out :) thanks!

**Review by katherinejensen00 — DISMISSED** [review]

**Pull request merged**